src/main/java/com/mixpanel/android/viewcrawler/ConnectivityIndicator.java

```diff
@@ -0,0 +1,201 @@
+package com.mixpanel.android.viewcrawler;
+
+import android.app.Activity;
+import android.content.Context;
+import android.graphics.PixelFormat;
+import android.os.IBinder;
+import android.util.DisplayMetrics;
+import android.util.Log;
+import android.view.Gravity;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewConfiguration;
+import android.view.WindowManager;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+
+import com.mixpanel.android.R;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConnectivityIndicator
```

Comment:
Have you tried to rotate the device while the indicator is on screen and move it around?